packages/glsp-client/package.json

```diff
@@ -32,8 +32,8 @@
     "@crossmodel/model-service": "0.2.5",
     "@crossmodel/protocol": "0.2.5",
     "@crossmodel/react-model-ui": "0.2.8",
-    "@eclipse-glsp/client": "next",
-    "@eclipse-glsp/theia-integration": "latest",
+    "@eclipse-glsp/client": "~2.4.0",
+    "@eclipse-glsp/theia-integration": "~2.4.0",
```

Comment:
Same as above, we should be using the theia-integration matching our Theia version.